plugins/scaffolder/src/index.ts

```diff
@@ -37,3 +37,5 @@ export {
   RepoUrlPicker,
   TextValuePicker,
 } from './components/fields';
+export { FavouriteTemplate } from './components/FavouriteTemplate';
+export { rootRouteRef as scaffolderRootRouteRef } from './routes';
```

Comment:
This should not be exported, it's used via `scaffolderPlugin.routes` instead

Reply:
Yeah your totally right. I address this on [22f18ff](https://github.com/backstage/backstage/pull/6829/commits/22f18ff0229ff65ba0a94e6358f75607e7d2d1b0)